app/helper/ethereum/helper.go

```diff
@@ -60,3 +62,24 @@ func EncodeData(ctm *proto.CryptoTransferMessage) ([]byte, error) {
 		amountBn,
 		feeBn)
 }
+
+func DecodeSignature(signature string) (decodedSignature []byte, ethSignature []byte, err error) {
```

Comment:
If this one is `DecodeSignature` should the one above be `EncodeSignature` instead of `EncodeData`?

Reply:
Encode data is used to rebuild the hash which is signed.